README.md

```diff
@@ -148,33 +161,6 @@ Elyra can be installed from PyPI:
 
 Run the following commands to verify the installation. Note that in the example output below the `[version]` placeholder is displayed instead of an actual version identifier, which might change with every release.
 
-```bash
-jupyter serverextension list
```

Comment:
We should remove this same output from https://github.com/elyra-ai/elyra/blob/master/docs/source/getting_started/installation.md#verify-installation

Reply:
done